src/build.rs

```diff
@@ -519,9 +535,66 @@ pub fn compile_options(
         FilterRule::none(),
     );
 
+    compile_opts.build_config.unit_graph = false;
+
     Ok(compile_opts)
 }
 
+#[derive(Default)]
+struct Exec {
+    ran: AtomicBool,
+    link_line: Mutex<String>,
```

Comment:
Should maybe also be an `Option` until it's actually set?

Reply:
I could have a single field thats Mutex<Option<String>> even if it is a bit more implicit maybe.

Reply:
I tried and the code gets more convoluted. :(

Reply:
Also the link line can be empty on purpose. So the code should stay this way.